memory_w25q1128jv_spi/memory_w25q1128jv_spi.h

```diff
@@ -0,0 +1,186 @@
+/*
+ * W25Q128JV_MEMORY.h
+ *
+ *  Created on: Jan 1, 2024
+ *      Author: rober
+ */
+#pragma once
+
+#include <cmath>
+
+#if defined(STM32F1)
+#include "stm32f1xx_hal.h"
+#endif
+
+#ifndef SPI_Timeout
```

Comment:
As per talk in PR #4, use unique timeout for each driver. use correct naming convention, suggest add prefix. `MEMORY_SPI_TIMEOUT` and remove `#ifndef` block

Reply:
sounds good. Im guessing this is an older comment and we're gonna do a `MEMORY_SPI_TIMEOUT` as a passed in, private variable?